packages/core/src/payment/strategies/bolt/bolt-script-loader.spec.ts

```diff
@@ -11,6 +11,15 @@ describe('BoltScriptLoader', () => {
     let scriptLoader: ScriptLoader;
     let mockWindow: BoltHostWindow;
 
+    const expectedScriptOptions = (id: string, publishableKey?: string) => {
```

Comment:
Do we need to move this to a function? Seems like can be part of the test itself.

Reply:
I move it to function to prevent multiple code duplication.
No logic inside this function, only constructor for data object with specific parameters, which different for different scripts but the same for all cases.